### PR TITLE
Fixes #140 -  CoinHours and calculatedHours conflict.

### LINKED ISSUE #140: Add ticker CalculatedHour to disambiguate between coinHours and calculatedHours in GetCoins method.
**Feature description**

To get the coins and coinHours in the app from a SkycoinTransactionOutput we call GetCoins with a Ticker that describe the type of value that is needed. Actually, the value calculatedHours is also needed from a output, but there is no way to get it from the implemented interfaces.

**Is your feature request related to a problem? Please describe.**
Where coinHours is requested it displays calculatedHours, this is a error.

**Describe the solution you'd like**
Add a Ticker named CalculatedHour and update GetCoins method.

**Possible implementation**

- [x] Add Ticker CalculatedHour in src/coin/skycoin/models/wallet.go.
- [x] Add a option to the ticker CalculatedHour and return the properly property
- [x] Fix option with Ticker CoinHour to return coinHours.
- [x] Finally review all the code to use the correct ticker in properly locations.

### PR DESCRIPTION
**Pull request by AntiD2ta**
Fixes #140

Changes:
- Add Ticker CalculatedHours for coin skycoin.
- Update code where GetCoins method is called.
- Update some models to use FormatCoins.

Requires testing
no

### CODE REVIEW

File: src/coin/skycoin/models/coin.go [hunk 1]
```diff
@@ -1,168 +1,169 @@
...
- 
-func (txn *SkycoinPendingTransaction) GetStatus() core.TransactionStatus { 
-  	return core.TXN_STATUS_PENDING
-} 
- 
-func (txn *SkycoinPendingTransaction) GetInputs() []core.TransactionInput { 
+}
+
+func (txn *SkycoinPendingTransaction) SupportedAssets() []string {
+	return []string{Sky, CoinHour}
```
Comment (olemis) [inline src/coin/skycoin/models/coin.go]: No calculated coin hours here ?

File: src/coin/skycoin/models/coin.go [hunk 2]
```diff
@@ -1,168 +1,168 @@
...
+func (txn *SkycoinPendingTransaction) GetId() string {
+	return txn.Transaction.Transaction.Hash
+}
+
+func (txn *SkycoinPendingTransaction) ComputeFee(ticker string) uint64 {
 	if ticker == Sky {
-		return uint64(0);
+		return uint64(0)
 	}
 	return txn.Transaction.Transaction.Fee
```
Comment (mauricio1802) [inline src/coin/skycoin/models/coin.go]: I think that an error should be returned if an invalid ticker is passed to the function

Comment (olemis) [reply src/coin/skycoin/models/coin.go]: 👍

Comment (olemis) [reply src/coin/skycoin/models/coin.go]: error instance can be created in code a this point , until error constant be added later to fix #143

**Review by olemis — APPROVED** [review]

**Pull request merged**